statemanager/statemanager.go

```diff
@@ -92,3 +93,21 @@ func (s *stateManager) SetStorage(address crypto.Address, key, value binary.Word
 
 	return err
 }
+
+// Bump the sequence number of an account
+func IncSequence(state StateManager, address crypto.Address) error {
+	const errHeader = "IncSequence():"
+	acc, err := state.GetAccount(address)
```

Comment:
with this we're back to a per-address increment.

Reply:
Hm not sure I follow, the behaviour is basically the same as:

https://github.com/hyperledger/fabric-chaincode-evm/pull/3/commits/3b2fd2b8490c5eafa13d0e0994536e3484f05d59#diff-072bdea284f7fd429bccb691f870c9ebL116